generated/nodejs/google-cloud-node/packages/errorreporting/package.json

```diff
@@ -3,19 +3,25 @@
   "name": "@google-cloud/errorreporting",
   "version": "0.7.1",
   "author": "Google Inc.",
-  "description": "Stackdriver Clouderrorreporting library for Node.js",
+  "description": "Errorreporting client for Node.js",
```

Comment:
This file is now auto generated and Errorreporting is the API name from the API config. If we need to add the Stackdriver we probably need to modify the api config file.